src/OnlineRetailPortal.Contracts/Contracts/UploadImageResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace OnlineRetailPortal.Contracts.Contracts
+{
+    public class UploadImageResponse
+    {
+        public int Code { get; set; }
+        public string Message { get; set; }
+
+        public IImageWriterResponse[] info;
+        public UploadImageResponse()
+        {
+            info = new IImageWriterResponse[1];
```

Comment:
please explain what are we trying to do here ??and where are you even initializing the info property